### PR TITLE
[MOB-5337] - InAppDisplayInterval resolution

### PR DESCRIPTION
**Pull request by Ayyanchira**
## 🔹 Jira Ticket(s)

* [MOB-5337](https://iterable.atlassian.net/browse/MOB-5337)

## ✏️ Description

1. Replace instances of retryInterval to moveToForegroundSyncInterval.
2. Map inAppDisplayInterval to moveToForegroundSyncInterval rather than converting to retryInterval

Context:
The configurable inAppDisplayInteval maps to variable - retryInterval which is used to check if its ready to show inapp after first inapp was displayed. However, the same property is not used when app switches from background to foreground. During this time, a variable - moveToForegroundSyncInterval is used which is redundant and is never changed.

### CODE REVIEW

File: swift-sdk/Internal/InAppManager.swift [hunk 1]
```diff
@@ -532,7 +531,7 @@ class InAppManager: NSObject, IterableInternalInAppManagerProtocol {
     
     private var syncResult: Pending<Bool, Error>?
     private var lastSyncTime: Date?
-    private let moveToForegroundSyncInterval: Double = 1.0 * 60.0 // don't sync within sixty seconds
+    private var moveToForegroundSyncInterval: Double = 1.0 * 60.0 // don't sync within sixty seconds
```
Comment (roninopf) [inline swift-sdk/Internal/InAppManager.swift:534]: Was this change needed?

**Review by roninopf — APPROVED** [review]

File: swift-sdk/Internal/InAppManager.swift (hunk 1 above)
Comment (Ayyanchira) [reply swift-sdk/Internal/InAppManager.swift:534]: Yes or the tests were failing when not being able to set a custom inapp interval
